docs/UserGuide.md

```diff
@@ -408,46 +409,86 @@ DELIVERY_DATE)!_
 
 **Command fails (invalid_date_format):** _Please provide the date in the format: yyyy-MM-dd._
 
-### View all deliveries
+### View a list of deliveries
 
-Shows a list of all deliveries.
+> Lists the deliveries in the delivery book.
 
-**Format:** `delivery list [--status STATUS] [--customer CUSTOMER_ID] [--date EXPECTED_DELIVERY_DATE]  [--sort SORT]`
+If you would like to get an overview of all your deliveries, you can use `delivery list`{.swift} to list all your
+deliveries at once. You can also filter the list of deliveries by status, customer ID and expected delivery date.
+You can also sort the list of deliveries by expected delivery date in ascending or descending order. By default, the
+list of deliveries will be sorted by expected delivery date in descending order (latest first).
```

Comment:
Will this be italicised?

Reply:
I think we shouldn't italicise this as we usually use italics to highlight or provide extra information / context.